.github/workflows/fairworkflows.yml

```diff
@@ -26,7 +26,8 @@ jobs:
           sudo apt-get install -y graphviz
       - name: Create RSA key for nanopubs
         run: |
-          make_nanopub_keys
+          # This will setup profile with a test orcid id, not publishing this-is-me to nanopub, and full name 'test'
+          printf '\nhttps://orcid.org/0000-0000-0000-0000\ntest\nn\n' | setup_profile
```

Comment:
OK, this does not make any sense... I should just pass these as arguments, there is no need to use the interactive prompt.

Reply:
Fixed, only for --keypair we cannot pass null values (or I could not find a way), so I had to use printf